.github/workflows/certora-prover.yml

```
@@ -0,0 +1,47 @@
+name: Certora Prover Workflow
+
+on:
+  push:
+    branches:
+      - certora
+      - master-rules
+  pull_request:
+    branches:
+      - master
+      - master-report
+      - certora-report
+  workflow_dispatch:
+
+jobs:
+  certora_run:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      statuses: write
+      pull-requests: write
+      id-token: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
+        with:
+          version: nightly
+      - name: Install forge dependencies
+        run: forge install
+      - name: Munge code
+        run: ./certora/scripts/munge.sh
+      - name: Run Certora pipeline
+        uses: Certora/certora-run-action@v2.3.0
+        with:
+          configurations: |-
+            certora/confs/CashModuleCore.conf --rule "p03"
+            certora/confs/EtherFiSafe.conf
+          solc-versions: 0.8.28
+          solc-remove-version-prefix: "0."
+          job-name: "EtherFi Contracts"
```

```suggestion
          job-name: "Certora verification"
```
Or something more descriptive, it's kind of obvious that we're working on EtherFi at this point 😄